.env.testing

```
@@ -0,0 +1,8 @@
+APP_ENV=testing
+DB_CONNECTION=sqlite
+DB_DATABASE=:memory:
+APP_KEY=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_KEY key should go before the DB_CONNECTION key

(UnorderedKey)